libraries/AP_Notify/ToneAlarm.cpp

```diff
@@ -145,6 +145,9 @@ void AP_ToneAlarm::play_tone(const uint8_t tone_index)
 {
     uint32_t tnow_ms = AP_HAL::millis();
     const Tone &tone_requested = _tones[tone_index];
+    
+    // Notification of alarm patterns to GCS
+    mavlink_msg_play_tune_send(mavlink_channel_t(0), 0, 0, nullptr, tone_requested.str);
```

Comment:
There's a few issues with this implementation.

First, this would really have to `gcs().send_message(MSG_PLAY_TUNE)` rather than attempting to send directly.

Secondly, pretty sure empty-string-rather-than-nullptr would be required.

Thirdly, we can't broadcast this.  In a swarm you'd end up every other drone in the swarm play your tune!  Perhaps we would only send to `SYSID_MYGCS`?